scanpy/plotting/_anndata.py

```diff
@@ -711,6 +711,8 @@ def violin(
     # of the violin plot (see stacked_violin code) for more info.
     if 'cut' not in kwds:
         kwds['cut'] = 0
+    if 'inner' not in kwds:
+        kwds['inner'] = None
```

Comment:
How about: `kwds.setdefault("inner")`, to skip the `if` statement?

Reply:
Do you mean `kwds.setdefault("inner", None)`?

Reply:
Oh default is None already, I didn't know that. ok, makes sense :)